### PR TITLE
Small refactoring around integration tests in config module

### PR DESCRIPTION
**Pull request by dst**

### CODE REVIEW

File: micro-infra-spring-config/src/test/groovy/com/ofg/infrastructure/property/decrypt/DecryptingPropertyExtendedSpec.groovy [hunk 1]
```diff
@@ -34,18 +33,17 @@ class DecryptingPropertyExtendedSpec extends AbstractIntegrationSpec {
...
         cleanup:
             context?.close()
     }
 
-    private static SpringApplicationBuilder defaultTestSpringApplicationBuilder() {
-        new SpringApplicationBuilder(DecryptingPropertyTestApp)
-                .web(false)
-                .showBanner(false)
+    private ConfigurableApplicationContext contextWithProperties(
+            String properties) {
```
Comment (szpak) [inline micro-infra-spring-config/src/test/groovy/com/ofg/infrastructure/property/decrypt/DecryptingPropertyExtendedSpec.groovy]: Why this line is broken?

File: micro-infra-spring-config/src/test/groovy/com/ofg/infrastructure/property/AbstractIntegrationSpec.groovy [hunk 2]
```diff
@@ -32,4 +34,16 @@ abstract class AbstractIntegrationSpec extends Specification {
     protected void setEncryptKey(String encryptKey = 'eKey') {
         System.setProperty(CLOUD_ENCRYPT_KEY, encryptKey)
     }
+
+    protected ConfigurableApplicationContext contextWithSources(
```
Comment (szpak) [inline micro-infra-spring-config/src/test/groovy/com/ofg/infrastructure/property/AbstractIntegrationSpec.groovy]: The same in that file.

File: micro-infra-spring-config/src/test/groovy/com/ofg/infrastructure/property/decrypt/DecryptingPropertySpec.groovy [hunk 3]
```diff
@@ -1,18 +1,13 @@
...
 import org.springframework.context.ConfigurableApplicationContext
 import org.springframework.context.annotation.Configuration
 import org.springframework.context.annotation.PropertySource
 import org.springframework.stereotype.Component
-import spock.lang.AutoCleanup
-import spock.lang.Ignore
-import spock.lang.IgnoreIf
-import spock.lang.Issue
-import spock.lang.Shared
+import spock.lang.*
```
Comment (szpak) [inline micro-infra-spring-config/src/test/groovy/com/ofg/infrastructure/property/decrypt/DecryptingPropertySpec.groovy]: AFAIK we prefer to not use `*` in imports.

**Comment by dst** [comment]
Fixed formatting issues.

**Pull request merged**